docs/data_model.md

```diff
@@ -1,19 +1,19 @@
-# odml data model
+# The odML data model
 
-Data exchange requires that also annoations, metadata, are
-exchanged. In oder to allow interoperability we need both a common
+Data exchange requires that also annotations, metadata, are
```

Comment:
This sentence isn't proper English (I know it's from before).  The 'also' doesn't work that way.

I guess what we're trying to say is
```suggestion
Data exchange requires that annotations as well as metadata are
```

Reply:
Or perhaps
```suggestion
Data exchange requires that annotations in addition to metadata are
```